groovy-language-server/src/main/java/com/palantir/ls/groovy/GroovyLanguageServer.java

```diff
@@ -61,7 +61,9 @@ public GroovyLanguageServer(CompilerWrapperProvider provider, LanguageServerConf
 
     @Override
     public CompletableFuture<InitializeResult> initialize(InitializeParams params) {
-        workspaceRoot = Paths.get(params.getRootPath()).toAbsolutePath().normalize();
+        Path tempWorkspaceRoot = Paths.get(params.getRootPath()).normalize();
```

Comment:
workspaceRoot = Paths.get(params.getRootPath()).toAbsolutePath().normalize()
should just work